js-fetch/js-fetch_star-wars-console/js/index.js

```diff
@@ -2,6 +2,29 @@ console.clear();
 
 const url = "https://swapi.py4e.com/api/people";
 
-function fetchData() {}
+async function fetchData() {
+    console.log("fetch started");
+    try{
+        const response = await fetch(url);
+        const data = await response.json();
+
```

Comment:
Also check for response not ok here

```
 if (!response.ok) {
      console.log("Response not ok");
      return;
    }
```

Reply:
True. To stop the application running, right?. Thanks Fadik.